.vscode/settings.json

```diff
@@ -10,11 +10,12 @@
     "packages/domain",
     "packages/eslint-config",
     "packages/prettier-config",
+    "packages/react-web",
     "packages/react",
     "packages/sdk-gated",
     "packages/shared-kernel",
     "packages/storage",
-    "packages/wagmi",
-    "packages/gated-content"
```

Comment:
is the removal of `"packages/gated-content"` intended here?

Reply:
nope...realized that and fixed in coming PR (latest of the series hopefully)